src/test/sql/sql_to_result_test.cpp

```diff
@@ -1,96 +0,0 @@
-#include <memory>
```

Comment:
Removed these since we have proper TPCH tests and these two queries tested here just don't fit in.